src/components/SignIn/index.js

```diff
@@ -1,20 +1,116 @@
-import React from 'react';
-
+import React, { Component } from 'react';
+import { withRouter } from 'react-router-dom';
+import { compose } from 'recompose';
+// import Footer from '../Footer';
 import { SignUpLink } from '../SignUp';
 import { PasswordForgotLink} from '../PasswordForget';
 
+import { withFirebase } from '../Firebase';
+import * as ROUTES from '../../constants/routes';
+
 import {SignInForm, SignInGoogleForm} from './SignInForm';
 
 const SignInPage = () => (
   <div>
+
+  
     <h1>SignIn</h1>
     <SignInForm />
     <SignInGoogleForm />
     <SignUpLink />
     <PasswordForgotLink/>
   </div>
 );
-
-
+//
```

Comment:
There is a SignInForm, and SignInGoogleForm in SignIn/SignInForm.js.
Change the current boostrap SignInForm to the new one